pkg/config/ext_authz.go

```diff
@@ -23,11 +23,11 @@ import (
 	"strconv"
 	"time"
 
-	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
-	core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
-	endpoint "github.com/envoyproxy/go-control-plane/envoy/api/v2/endpoint"
-	extAuthService "github.com/envoyproxy/go-control-plane/envoy/config/filter/http/ext_authz/v2"
-	httpconnectionmanagerv2 "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/http_connection_manager/v2"
+	v3Cluster "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
```

Comment:
nit: Let's just make this `cluster`

Reply:
Oh yes.. I still need to clean all of that.. don't review it yet ;)